scalameta/tokenizers/shared/src/main/scala/scala/meta/internal/tokenizers/XmlParser.scala

```diff
@@ -11,124 +11,114 @@ import fastparse.all._
 
 class XmlParser(Block: P0,
                 Patterns: P0 = Fail,
-                WL: P0 = CharIn("\n ").rep(),
```

Comment:
We actually have close to no tests for xml literals. Any chance we could add a few positive and negative tests? Something like

```scala
checkOk("<...>")
checkFail("<...>")
```
That would be very valuable to guard against future changes breaking the xml parser.